_posts/2019-11-08-PC-Engines-Firmware-v4-10-0-3.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.10.0.3"
+date:   2019-11-08
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.10.0.3
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** 2d90cb1.
+
```

Comment:
@piotr-kleins please add some info that we added ZIP packages with binaries for Windows users.